### DISCUSSION TITLE
Add reports from Secure3 Audit platform

### DISCUSSION THREAD

**Discussion by Robert-H-Leonard** [solodit/feedback#41] [New Feature Requests]
Secure3 is an audit contest platform that has done hundreds of audits with over 50 of them with public reports: https://github.com/Secure3Audit/Secure3Academy

Some of the security researchers finding issues in this contest are also very active on code4 and sherlock and there is a lot of good knowledge to be learned.

**Comment by hans-cyfrin** [comment]
Thanks for the suggestion. 
Reports can be added by adding a PR on [solodit_content](https://github.com/solodit/solodit_content) repository.
Checking Secure3 reports, we see that all reports are in PDF formats and it would be challenging because it would need a custom parsing script.
With some other features in progress (that I believe are more useful and necessary), we don't have the capacity.
We have a guide on [Notion](https://notion.so/EXTERNAL-Instructions-for-Audit-firms-to-add-reports-on-Solodit-4edb6b07b3fb4d3ea3c7b875a9da32ea) if anyone is willing to write a script to parse the reports.
